static/styles/right-sidebar.css

```diff
@@ -197,35 +197,3 @@
 .add-user-list-filter {
     width: 80%;
 }
-
-#user-list .input-append {
-    margin-bottom: 5px;
```

Comment:
It looks like the `margin-bottom` property wasn't moved as part of this commit.  Your commit message should explain why this wasn't moved.